### PR TITLE
Circle layer example update

### PR DESCRIPTION
**Pull request by tobrun**
Update to our Circle layer to investigate a bug.

ezgif com-video-to-gif 1

### CODE REVIEW

File: platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/CircleLayerActivity.java [hunk 1]
```diff
@@ -57,40 +58,50 @@ public void onCreate(Bundle savedInstanceState) {
             @Override
             public void onMapReady(@NonNull final MapboxMap map) {
                 mapboxMap = map;
+                try {
+                    mapboxMap.addSource(new GeoJsonSource("bus_stop", new URL("https://raw.githubusercontent.com/cheeaun/busrouter-sg/master/data/2/bus-stops.geojson")));
+                } catch (MalformedURLException malformedUrlException) {
+                    Log.e(
```
Comment (cammace) [inline platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/CircleLayerActivity.java]: Timber?

**Comment by zugaldia** [comment]
@tobrun Do you need my 👀 here?

**Comment by tobrun** [comment]
Putting on my list to fix conflicting files, will ping you for review.

**Comment by zugaldia** [comment]
@tobrun Sounds like a plan.

**Comment by tobrun** [comment]
@zugaldia rebased, resolved and fixed up @cammace comment.

**Comment by zugaldia** [comment]
@cammace could you 👍 / 👎 ?

**Pull request merged**